easybuild/easyconfigs/f/fermi-lite/fermi-lite-20190320-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,44 @@
+# Author: Jasper Grimm (UoY)
+
+easyblock = 'MakeCp'
+
+name = 'fermi-lite'
+version = '20190320'
+local_commit = 'b499514'
+
+homepage = 'https://github.com/lh3/fermi-lite'
+description = """Standalone C library for assembling Illumina short reads in small regions."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+github_account = 'lh3'
+source_urls = [GITHUB_SOURCE]
+sources = ['%s.tar.gz' % local_commit]
+patches = [
+    '%(name)s-20190320_avoid-bwa-namespace-conflict.patch',
+    '%(name)s-20190320_build-shared-lib.patch',
+]
+checksums = [
+    'ea1230e298b8e7193a996b5aceeff7fea44ef00852b0b87d4fcb504bdca0e712',  # b499514.tar.gz
+    # fermi-lite-20190320_avoid-bwa-namespace-conflict.patch
+    '893f7489283a85cb0dabf2da71f10fc4527fbfe2b110d8618293732fa74eabab',
+    '46978098bb439d4772f263375eb67621489bb48853c182d7aab9b9da8a41494c',  # fermi-lite-20190320_build-shared-lib.patch
+]
+
+builddependencies = [('binutils', '2.34')]
+
+dependencies = [('zlib', '1.2.11')]
+
+files_to_copy = [
+    (['fml-asm'], 'bin'),
+    (['*.a', '*.so*'], 'lib'),
+    (['*.h'], 'include/fml'),
+    'test',
+]
+
+sanity_check_paths = {
+    'files': ['bin/fml-asm', 'lib/libfml.a', 'lib/libfml.%s' % SHLIB_EXT],
+    'dirs': [],
+}
+
```

Comment:
If possible, can you add a sanity_check_commands that does at least something like `fml-asm --version` or something?

Reply:
Unfortunately, `fml-asm` doesn't look for `--version`, `--help` or similar, and providing no arguments prints the usage but exits with status code 1